config/sync/simple_oauth.settings.yml

```diff
@@ -0,0 +1,11 @@
+_core:
+  default_config_hash: KsPFWSp6mgXIQgjBJEShfKUGn6VLRlbpIJ2EysXvXWM
+access_token_expiration: 300
+authorization_code_expiration: 300
+refresh_token_expiration: 1209600
+token_cron_batch_size: 0
+public_key: ../public.key
+private_key: ../private.key
```

Comment:
Are these keys that we need to ensure being present?

Reply:
Yes, we need those 2 keys to be present. 
However I am not sure what the best process for doing so is. Also I am a bit unsure on which directory they might best be stored.

Reply:
I have now added functionality that generates the keys and updated the configuration to look for the keys in the right direcotry.